### ISSUE TITLE
Dspot tests are failing 

### ISSUE THREAD

**Issue by borisbaldassari** [STAMP-project/dspot#803]
#### Characteristics
- **Issue Type**: test report
- **Reproducibility**: sometimes
- **Severity**: minor
- **Tool/Service/Component**: Dspot
- **Execution Environment**: Debian 9.8, JDK Oracle 1.8.0_201
- **Reporter**: Boris Baldassari

#### Description

When trying to compile and test dspot I encounter the following exception, which causes the build to fail.

#### Steps to reproduce

```
$ mvn clean package
[SNIP]
Results :

Tests in error: 
  AutomaticBuilderHelperTest.testGetFilterNoFilterSpecifiedAndClassInDefaultPackage:50 NullPointer
  AutomaticBuilderHelperTest.tearDown:56 NullPointer

Tests run: 194, Failures: 0, Errors: 2, Skipped: 6

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] DSpot - Parent ..................................... SUCCESS [  0.169 s]
[INFO] DSpot - Core ....................................... FAILURE [04:26 min]
[INFO] DSpot - Maven Plugin ............................... SKIPPED
[INFO] DSpot - Prettifier ................................. SKIPPED
[INFO] DSpot - Diff Test Selection ........................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 04:27 min
[INFO] Finished at: 2019-05-24T21:50:02+02:00
[INFO] Final Memory: 37M/806M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project dspot: There are test failures.
```

Please see the attached log for more details.

[dspot_clean_package.log.zip](https://github.com/STAMP-project/dspot/files/3218467/dspot_clean_package.log.zip)

**Comment by danglotb** [comment]
Hello @borisbaldassari 

Thank you for reporting this.

#806 should fix this. Please reopen this issue if you still experience this bug after the merge.

If you do plan to use only DSpot, I advise you to use directly the released jar or the maven plugin.

**Comment by borisbaldassari** [comment]
Thanks for the reply @danglotb 
I'll wait for the merge, give it a try, and close the issue if it's fixed.

**Comment by danglotb** [comment]
I just merged the #806

**Comment by borisbaldassari** [comment]
Works for me. Thanks @danglotb!
